db.go

```diff
@@ -877,7 +877,7 @@ func (db *DB) openHeadBlock(dir string) (*HeadBlock, error) {
 	)
 	wal, err := OpenSegmentWAL(wdir, l, 5*time.Second)
 	if err != nil {
-		return nil, errors.Wrap(err, "open WAL %s")
+		return nil, errors.Wrap(err, "open WAL %s", dir)
```

Comment:
Thanks, but I think it should be `errors.Wrapf()`. Wrap doesn't take the extra formatting parameters and hence the build is failing, moving to Wrapf should fix it.